app/services/setting_registry.rb

```diff
@@ -89,10 +89,9 @@ def set_user_value(name, value)
 
   # Returns all the categories used for settings
   def categories
-    sticked_general = { 'general' => nil }
-    sticked_general.merge!(Hash[@settings.map { |_name, definition| [definition.category_name, definition.category_label] }])
-    sticked_general.delete('general') if sticked_general['general'].nil?
-    sticked_general
+    @categories ||= @settings.values.uniq(&:category).each_with_object({'general' => nil}){ |definition, memo|
```

Comment:
We already have `known_categories`. What's the difference?

Reply:
`known_categories` are the old (DB) categories.